Documentation/security/threat-model.rst

```diff
@@ -0,0 +1,751 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Threat Model
+============
+
+In this section, we present a threat model for Cilium. This threat model
+allows interested parties to understand:
+
+-  security-specific implications of Cilium’s architecture
+-  controls that are in place to secure data flowing through Cilium’s various components
+-  recommended controls for running Cilium in a production environment
+
+Scope and Prerequisites
+-----------------------
+
+This threat model considers the possible attacks that could affect an
+up-to-date version of Cilium running in a production environment; it
+will be refreshed when there are significant changes to Cilium’s
+architecture or security posture.
+
+This model does not consider supply-chain attacks, such as attacks where
+a malicious contributor is able to intentionally inject vulnerable code
+into Cilium. For users who are concerned about supply-chain attacks,
+Cilium’s `recent security audit`_ assessed Cilium’s supply chain controls
+against `the SLSA framework`_.
+
+In order to understand the following threat model, readers will need
+familiarity with basic Kubernetes concepts, as well as a high-level
+understanding of Cilium’s `architecture and components`_.
+
+.. _recent security audit: https://github.com/cilium/cilium.io/blob/main/Security-Reports/CiliumSecurityAudit2022.pdf
+.. _the SLSA framework:  https://slsa.dev/
+.. _architecture and components: https://docs.cilium.io/en/stable/overview/component-overview/
```

Comment:
It's best to avoid full internal links in the tree, since we end up maintaining multiple versions, and this can inadvertently lead users who are reading one version of the docs into another different version of the docs and confuse them.

I'd suggest using internal anchors + `:ref:` instead - ie, apply this diff on top of the branch:

```
diff --git a/Documentation/overview/component-overview.rst b/Documentation/overview/component-overview.rst
index 3ef1f05e511a..7c8ce2809cf6 100644
--- a/Documentation/overview/component-overview.rst
+++ b/Documentation/overview/component-overview.rst
@@ -4,6 +4,8 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. _component_overview:
+
 ******************
 Component Overview
 ******************
diff --git a/Documentation/security/threat-model.rst b/Documentation/security/threat-model.rst
index 33b3fb0b8344..bfbe4b4788fe 100644
--- a/Documentation/security/threat-model.rst
+++ b/Documentation/security/threat-model.rst
@@ -30,11 +30,10 @@ against `the SLSA framework`_.
 
 In order to understand the following threat model, readers will need
 familiarity with basic Kubernetes concepts, as well as a high-level
-understanding of Cilium’s `architecture and components`_.
+understanding of Cilium’s :ref:`architecture and components<component_overview>`.
 
 .. _recent security audit: https://github.com/cilium/cilium.io/blob/main/Security-Reports/CiliumSecurityAudit2022.pdf
 .. _the SLSA framework:  https://slsa.dev/
-.. _architecture and components: https://docs.cilium.io/en/stable/overview/component-overview/
 
 Methodology
 -----------
```